src/encoded/static/components/experiment.js

```diff
@@ -741,6 +730,31 @@ const replicateTableColumns = {
         },
     },
 
+    genetic_modification: {
+        title: 'Modifications',
+        display: (condensedReplicate) => {
+            const replicate = condensedReplicate[0];
+            const gms = replicate.library && replicate.library.biosample && replicate.library.biosample.applied_modifications;
```

Comment:
not that this is going to effect the request, but man JS is clunky here.  So is python I guess.